calico-cloud/reference/resources/runtimesecurity.mdx

```diff
@@ -0,0 +1,109 @@
+---
+description: API for this Calico Cloud resource 
+---
+
+# RuntimeSecurity
+
+The **RuntimeSecurity** custom resource (CR) is used to enable and configure Container Threat Detection in a Calico Cloud managed cluster.
+
+### Resource Definition
+
+```yaml
+apiVersion: operator.tigera.io/v1
+kind: RuntimeSecurity
+metadata:
+  name: defaultmy-runtimesecurity-config
+spec:
+  detectorConfig:
+    - id: 190
+      disabled: false
+    - id: 330
+      disbaled: True
+  runtimeExceptionList:
+    - matching: regex
+      processInvocation: "/bin/ls*"
+      pod: "not-evil-pod"
+      namespace: "default"
+    - matching: exact
+      pod: "nginx"
+      namespace: default
+    - matching:
+      namespace: "company-operations"
+```
+
+## Runtime Security Definition 
+
+### Metadata
+
+| Field  | Description                                   | Accepted Values                           | Schema |
+| ------ | --------------------------------------------- | ----------------------------------------- | ------ |
+| name   | The name of the runtime security resource.    | Lower-case alphanumeric with optional `-` | string |
+| labels | A set of labels to apply to this resource.    |                                           | map    |
+
+
+### Spec
+
+| Field                    | Description                                                                                  | Accepted Values  | Schema                                                | Default |
+| ------------------------ | -------------------------------------------------------------------------------------------- | ---------------- | ----------------------------------------------------- | ------- |
+| detectorConfig           | Configuration parameters for a detector in a Calico Cloud managed cluster.                   |                  | [DetectorConfig](#detectorconfig)                     |         |
+| runtimeExceptionList     | List of entries of suspicious processes that are allowed to run that wont generate an event  |                  | [runtimeExceptionList](#runtimeexceptionlist)         | Enabled |
```

Comment:
sp: won't

Reply:
I think should remove "suspicious" here.

Reply:
my thinking was there are lots of processes that don't generate an event so we wanted to specify that it is meant to suspicious process. but happy to remove it all the same :slightly_smiling_face: